contracts/upgradeable_contracts/arbitrary_message/BasicHomeAMB.sol

```diff
@@ -37,6 +37,16 @@ contract BasicHomeAMB is BasicAMB, MessageDelivery {
         }
     }
 
+    /**
+    * @dev Requests message relay to the opposite network, message is sent to the manual lane.
+    * @param _contract executor address on the other side.
+    * @param _data calldata passed to the executor on the other side.
+    * @param _gas gas limit used on the other network for executing a message.
+    */
+    function requireToConfirmMessage(address _contract, bytes _data, uint256 _gas) external returns (bytes32) {
+        return _sendMessage(_contract, _data, _gas, 0xf0);
```

Comment:
does it make sense to introduce constants for `0xf0` and `0` used for the `_sendMessage` call?